packages/mobile/src/analytics/Events.tsx

```diff
@@ -29,6 +29,7 @@ export enum HomeEvents {
   notification_select = 'notification_select',
   transaction_feed_item_select = 'transaction_feed_item_select',
   transaction_feed_address_copy = 'transaction_feed_address_copy',
+  impression_cash_in_bottom_sheet = 'impression_cash_in_bottom_sheet',
```

Comment:
Nit: I'd move this to the `FiatExchangeEvents` enum and rename this something like `cico_add_funds_bottom_sheet_impression`.

While this event is fired on the WalletHomeScreen, I think `FiatExchangeEvents` is the better logical grouping.

Also, if we start all events related to the cash-in buttons with `cico_add_funds`, then that will ensure those tables are next to each other in the database as tables are ordered alphabetically.